cs/src/core/Index/Common/Contexts.cs

```diff
@@ -38,15 +38,17 @@ internal class SerializedFasterExecutionContext
     {
         internal long version;
         internal long serialNum;
-        internal string guid;
+        internal int sessionID;
+        internal string sessionName;
 
         /// <summary>
         /// </summary>
         /// <param name="writer"></param>
         public void Write(StreamWriter writer)
         {
             writer.WriteLine(version);
-            writer.WriteLine(guid);
+            writer.WriteLine(sessionName);
```

Comment:
Does this work correctly with null sessionNames?

Reply:
fixed